src/Microsoft.Azure.WebJobs.Host/Executors/FunctionInstanceFactory.cs

```diff
@@ -21,10 +20,10 @@ public FunctionInstanceFactory(IFunctionBinding binding, IFunctionInvoker invoke
             _descriptor = descriptor;
         }
 
-        public IFunctionInstance Create(Guid id, Guid? parentId, ExecutionReason reason, IDictionary<string, object> parameters)
+        public IFunctionInstance Create(FunctionInstanceFactoryContext context)
```

Comment:
The main change is just an internal refactoring to use a context object here, which is a good change to make anyways